.github/workflows/go.yml

```diff
@@ -0,0 +1,88 @@
+name: CI
+
+on:
+  schedule:
+    # run every Sunday to re-populate caches after they are cleaned on Saturday
+    - cron: "0 12 * * 0"
+  push:
+    branches:
+      - main
+      - release/*
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        go-version:
+          - 1.15.x
+        may-fail: [false]
+        
+        include:
+          - go-version: tip
+            may-fail: true
+    continue-on-error: ${{ matrix.may-fail }}
+
+    env:
+      GOPATH: ${{ github.workspace }}
+      WORKING_DIR: ${{ github.workspace }}/src/github.com/percona/pmm-agent
+
+    defaults:
+      run:
+        working-directory: ${{ env.WORKING_DIR }}
+    steps:
+    - name: Check out code into the Go module directory
+      uses: percona-platform/checkout@v2
+      with:
+        fetch-depth: 1
+        path: ${{ env.WORKING_DIR }}
+
+    - name: Set up Go release
+      if: matrix.go-version != 'tip'
+      env:
+        # to avoid error due to `go version` accepting -v flag with an argument since 1.15
+        GOFLAGS: ""
```

Comment:
We don't define `GOFLAGS` in global `env:` above (with `GOPATH` and `WORKING_DIR`), so this can be removed